relay-general/src/protocol/stacktrace.rs

```diff
@@ -126,6 +126,9 @@ pub struct Frame {
     /// then symbolication can take place.
     pub instruction_addr: Annotated<Addr>,
 
+    /// Defines the addressing mode for addresses.
+    pub addr_mode: Annotated<String>,
```

Comment:
We should consider validating the format here, but I think it's fine to merge as string.